Task 1/index.js

```diff
@@ -0,0 +1,48 @@
+/* 
+Дан масив об'єктів. Вивести масив телефонних номерів користувачів, у яких баланс більше 2000 доларів. 
+І знайти суму всіх балансів користувачів
+*/
+
+import users from './data/data.json' assert { type: 'json' };
+import { transformAndDeleteCharacters } from './helpers/helper.js';
+
+const root = document.querySelector('.root');
+
+const findBalanceMore2k = (value) => {
+	const generateFilterArr = value.filter((item) => {
+		if (transformAndDeleteCharacters(item.balance) >= 2000) {
+			return item;
+		}
+	});
+
+	return generateFilterArr;
+};
+
+const showPhoneUsers = () => {
+	const arr = findBalanceMore2k(users);
+	return arr.map((item) => {
+		const div = document.createElement('div');
+		div.classList.add('phone');
+		div.innerHTML = item.phone;
+		root.append(div);
+	});
+};
```

Comment:
this function useless. Remember - function should do only one thing - either calculate/generate/... a value, either render in html (sent on backend, saved on document, etc..). Imagine that your result calculated  in filterUserPhonesByBalance function should be sent on backend, or should be saved on some document.